pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -20,13 +22,13 @@
 
 buildPythonPackage rec {
   pname = "ansible-lint";
-  version = "6.4.0";
+  version = "6.7.0";
   format = "pyproject";
   disabled = pythonOlder "3.6";
```

Comment:
This is incidental, so from my point of view you don't have to address it in this PR, but I saw that, from 6.0.0 onwards, ansible-lint only formally supports Python versions 3.8 and greater (https://github.com/ansible/ansible-lint/pull/1850).